hw/ip/otbn/doc/_index.md

```diff
@@ -265,6 +285,27 @@ CSRs can be accessed through dedicated instructions, `CSRRS` and `CSRRW`.
       <td>
         <strong>RND</strong>.
         A random number.
+        The number is sourced from the EDN via a single-entry cache.
+        Reads when the cache is empty will cause OTBN to be stalled until a new random number is fetched from the EDN.
```

Comment:
```suggestion
        Reads when the cache is empty will cause OTBN to be stalled until a new random number is available.
```

The EDN is mostly a transmission network, the delay mostly comes from the physical noise source, the checking, conditioning, etc. I don't think that matters too much here, allowing us to keep it vague and focused on the behavior observed by the software developer.